inc/saischeduler.h

```diff
@@ -71,7 +71,7 @@ typedef enum _sai_scheduler_attr_t
     /**
      * @brief Scheduling algorithm weight
      *
-     * Range [1 - 100].
+     * Range [0 - 255], 0-invalid weight
```

Comment:
hi Kumaresh, will you also update the sonic-mgmt test to cover the higher values?